website/views/organization.py

```
@@ -3984,3 +4002,199 @@ def extract_payment_info_from_comments(self, issue, owner, repo_name, issue_numb
+            {
+                "error": f"Slack API error: {e.response.get('error')}",
+                "organization": organization,
+            },
+        )
+    except SlackRequestError as e:
+        # Network/connection/invalid response issues
+        return render(
+            request,
+            template,
+            {
+                "error": f"Slack network error: {str(e)}",
```

Ruff (0.14.5)

Use explicit conversion flag

Replace with conversion flag

(RUF010)